crates/host/src/lib.rs

```diff
@@ -1,39 +1,48 @@
-//! wasmCloud host library
-
+#![doc = include_str!("../README.md")]
 #![warn(missing_docs)]
 #![forbid(clippy::unwrap_used)]
 
-/// wasmbus host
-pub mod wasmbus;
+/// [crate::config::ConfigManager] trait for managing a config store which can be watched to receive
```

Comment:
Nit: I feel like this style of documenting these modules is a little odd. It is describing it primarily as if it were the trait, and it also is all in this top level. I feel like it would be cleaner to put these as module level comments in the module files themselves and to reword such that it is describing the module and then saying something like "the main feature of this module is the [crate::config::ConfigManager] trait..."